src/hooks/UrqlUseSubscriptionWithHandler.re

```diff
@@ -48,25 +52,28 @@ let useSubscriptionResponseToRecordWithHandler =
   {fetching, data, error, response};
 };
 
-let useSubscriptionWithHandler =
+let useSubscription =
     (
-      ~request: UrqlTypes.request('response),
-      ~handler: UrqlTypes.parsedHandler('acc, 'response),
-    ) => {
+      type a, type b, type c,
+      ~handler: handler(a, b, c),
+      request: UrqlTypes.request(b),
```

Comment:
Can you explain the logic around supplying `~handler` as a named function parameter and leaving `request` unnamed? Just not super familiar with this style 😅 and I'd love to understand the reasoning!

Reply:
At first I went with handler as a named argument so that later we could make it optional, but after looking into it, it seems like we won't be able to make it optional so we can change it to a normal argument